docs/jsonrpc_apis.md

```diff
@@ -388,6 +388,33 @@ result sample:
 }
 ```
 
+#### getreceivedbyaddress
```

Comment:
According to the description, this interface returns the balance of an address, getaccountbalance is more proper.

Reply:
bitcoin core apis has no interface named "getaccountbalance", but it has an interface named "getreceivedbyaddress".